pkg/services/sqlstore/org.go

```diff
@@ -122,6 +122,13 @@ func CreateOrg(cmd *m.CreateOrgCommand) error {
 func UpdateOrg(cmd *m.UpdateOrgCommand) error {
 	return inTransaction2(func(sess *session) error {
 
+		orgQuery := &m.GetOrgByIdQuery{Id: cmd.OrgId}
+
+		orgDoesntExists := GetOrgById(orgQuery)
```

Comment:
How do we know which databases support it? 

I guess we could write a uni test that confirms it and make sure to run it on all databases that we support. (as we should regardless) :)